docs/contributing.md

```diff
@@ -0,0 +1,30 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Contributing to Submariner](#contributing-to-submariner)
+  - [Overview](#overview)
+  - [How to contribute](#how-to-contribute)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing to Submariner
+
+:+1::tada: First off, thanks for taking the time to contribute to Submariner! :tada::+1:
+
+## Overview
+
+Submariner is a tool built to connect overlay networks of different Kubernetes clusters. It is CNI agnostic.
```

Comment:
It is CNI agnostic -> not completely true.

I'd say. It aims to be CNI agnostic.

Reply:
Done